### PR TITLE
fix: update logo in index template

### PR DESCRIPTION
**Pull request by warjiang**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
update the default logo for the dashboard

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
- update the default logo for the dashboard
```

### CODE REVIEW

File: ui/apps/dashboard/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```
Comment (RainbowMango) [inline ui/apps/dashboard/index.html:5]: Do we need to remove the `vite.svg` from code base?

Comment (warjiang) [reply ui/apps/dashboard/index.html:5]: yes ~

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**